tools/cli/clusterCommands.go

```diff
@@ -25,11 +25,17 @@
 package cli
 
 import (
+	"fmt"
 	"os"
 	"sort"
 
 	"github.com/olekukonko/tablewriter"
 	"github.com/urfave/cli"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+const (
+	fullWorkflowServiceName = "temporal.api.workflowservice.v1.WorkflowService"
```

Comment:
This string is hardcoded on server also. I don't want to reference it due to keeping loose coupling between `tctl` and server.